composer.json

```diff
@@ -12,14 +12,14 @@
         { "name": "Jeremy Mikola", "email": "jmikola@gmail.com" }
     ],
     "require": {
-        "php": "^7.1",
-        "symfony/framework-bundle": "^3.4|^4",
-        "symfony/console": "^3.4|^4",
-        "symfony/dependency-injection": "^3.4|^4",
-        "symfony/property-access": "^3.4|^4",
-        "pagerfanta/pagerfanta": "^1.0.5|^2.0",
+        "php": "^7.2",
+        "symfony/framework-bundle": "^4.3",
```

Comment:
As @OskarStark already stated, we have the possibility to introduce breaking changes with the bump to version 6 of the bundle, so it will be simpler to require Symfony and discard support for Symfony 3.4

Reply:
That's fine :)

Reply:
I'm a little overbooked for the moment.
I also made a lot of PR to ruflin/elastica, but I should be able to continue working on it soon.
I already use this PR as a patch for supporting ES 7.0 on our production servers ;)